drivers/avr/ssd1306.h

```diff
@@ -52,7 +52,7 @@ enum ssd1306_cmds {
 #define SSD1306_ADDRESS 0x3C
 #endif
 
-#define DisplayHeight 32
+#define DisplayHeight 64
```

Comment:
In its current form, this will break behaviour for other keyboards.

it either needs to be reverted, or wrapped with a ifndef and overridden at the keyboard level.